ee/keys_ee.go

```diff
@@ -47,6 +47,16 @@ func GetKeys(config *viper.Viper) (*Keys, error) {
 		return nil, fmt.Errorf(
 			"ACL secret key must have length of at least 32 bytes, got %d bytes instead", l)
 	}
+	// read public key after the length test because public key can be longer - no limit
+	aclPublicKeyFile := aclSuperFlag.GetPath(flagAclKeyFile)
```

Comment:
I am a little confused, how are we using the same key as public key and also as private key.